.codespellrc

```diff
@@ -1,2 +1,3 @@
 [codespell]
-skip = libtool,./autom4te.cache/*,*.html,./build-aux/*,./.git/*,aclocal*,./m4/*,./config.status,./configure,./config.log,todo
+skip = libtool,./autom4te.cache/*,*.html,./build-aux/*,./.git/*,aclocal*,./m4/*,./config.status,./configure,./config.log,todo,jquery.js
```

Comment:
```suggestion
skip = libtool,./autom4te.cache/*,*.html,./build-aux/*,./.git/*,aclocal*,./m4/*,./config.status,./configure,./config.log,todo,jquery.js,*~
```

it should also skip backup files like `configure~`

Reply:
I have implemented this change in the new force push.